source/modules/collection-api/user/overview.md.erb

```diff
@@ -0,0 +1,23 @@
+## Account Activity
+
+You can send user-related information to Inspetor via the REST API. The `/user` endpoint supports `POST` (create), `PUT` (update), and `DELETE` (delete) operations with the following arguments:
+
+Property               | When is it required?   | Type                  | Description
+---------------------- | ---------------------  | --------------------- | -----------
+id                     | `POST`, `PUT`, `DELETE`| String                | Your unique identifier for this user in your platform
+email                  | `POST`                 | String                | The email associated with the user
+document               |                        | String                | The Document number associated with the user.
+document_type          |                        | String                | The Document type associated with the user's document number. Accpeted values are `cpf`, `cnpj`, `passport`, `rg`, and `other`.
```

Comment:
Any reason why Document is capitalized in this line's and the one above's Description fields?

Reply:
Nope!